sdk/eventhub/Azure.Messaging.EventHubs/src/Amqp/AmqpClient.cs

```diff
@@ -545,6 +545,35 @@ public override async Task CloseAsync(CancellationToken cancellationToken)
             }
         }
 
+        /// <summary>
+        ///   Creates the AMQP link to be used for management operations and ensures
+        ///   that any corresponding state has been updated based on the link configuration.
+        /// </summary>
+        ///
+        /// <param name="operationTimeout">The timeout to apply to management operations using the link..</param>
+        /// <param name="linkTimeout">The timeout to apply for creating the link.</param>
+        /// <param name="cancellationToken">The cancellation token to consider when creating the link.</param>
+        ///
+        /// <returns>The AMQP link to use for management operations.</returns>
+        ///
+        private async Task<RequestResponseAmqpLink> CreateManagementLinkAsync(TimeSpan operationTimeout,
+                                                                              TimeSpan linkTimeout,
+                                                                              CancellationToken cancellationToken)
+        {
+            var link = default(RequestResponseAmqpLink);
+
+            try
+            {
+                link = await ConnectionScope.OpenManagementLinkAsync(operationTimeout, linkTimeout, CancellationToken.None).ConfigureAwait(false);
+            }
+            catch (Exception ex)
```

Comment:
Unfortunately, due to the inability to mock `Microsoft.Azure.Amqp` constructs or deterministically trigger the exception in live calls, there is no way to have a test validate.